tools/sggosemanticrelease/tools.go

```diff
@@ -32,6 +33,7 @@ func ReleaseFromCloudBuildCommand(ctx context.Context, ci bool, repo string) *ex
 		"--provider-opt",
 	}
 	args = append(args, "slug="+repo)
```

Comment:
Just a thought, I think the `github-prover` requires the `slug` option, so perhaps include it in the "default args" list above?

Reply:
I don't know enough about to say